.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,119 @@
+name: CI-PR
+on:
+  pull_request:
+    branches: [master]
+concurrency:
+  group: ci-${{ github.head_ref }}
+  cancel-in-progress: true
+jobs:
+  ci-pr:
+    name: CI-PR
+    needs: [test, msrv, lint]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Done
+        run: exit 0
+  test:
+    name: Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "windows-latest"]
```

Comment:
You don't need the quotes. Let's remove them to conform with the YAML style we use everywhere.

Reply:
Updating.  Personally, I've run into enough issues with quoting and yaml, I tend to default to quoting.